server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -226,10 +227,10 @@ protected void parseConfiguration(ServerConfiguration conf) {
     if (null == t) {
       throw new RuntimeException("Table Operations cannot be null");
     }
-    tableIdToTableName = new HashMap<>();
-    poolNameToRegexPattern = new HashMap<>();
+    Map<String,String> tableIdToTableNameBuilder = new HashMap<>();
```

Comment:
Could use an ImmutableMap.Builder

Reply:
I tried that originally, but it contains nulls?  Or maybe it was duplicate keys.  I patched this a while ago but failed to contribute it back at the time.

Reply:
I have run into those issues with Guava's immutable code before.  Your change seems like the best approach with those issues.